### PR TITLE
Added support for github tokens

### PR DESCRIPTION
**Pull request by creatorrr**
Allows loading files from private repos if `GITHUB_TOKEN` environment variable is set.

### CODE REVIEW

File: packyou/py3.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 from importlib.abc import MetaPathFinder
 
-from packyou import find_module_in_cloned_report
```
Comment (llazzaro) [inline packyou/py3.py]: haha! I was working with reports. thanks for fixing this typo

Comment (creatorrr) [reply packyou/py3.py]: hehe yeah, any eta on cutting the new `0.1.6` release to `pypi`? I would love to help push that

Comment (llazzaro) [reply packyou/py3.py]: are you using python 2?

Comment (creatorrr) [reply packyou/py3.py]: nope

**Pull request merged**